### PR TITLE
Add Gelma's typos from "n" to "o"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -38625,6 +38721,12 @@ ownder->owner
 ownerhsip->ownership
 ownersip->ownership
 ownes->owns, ones,
+ownload->download, own load,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38724]: Perhaps `unload`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -36504,6 +36522,12 @@ nighlties->nightlies
 nighlty->nightly
 nighther->neither
 nightime->nighttime
+nigth->night
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:36525]: I'll add `nigh`. I couldn't find `ninth`.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:38724]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -36488,7 +36503,10 @@ newslines->newlines
 newthon->newton
 newtork->network
 Newyorker->New Yorker
+nexted->nested, texted,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:36506]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**